<launch>
  <node pkg="hba" type="visualize" name="visualize" output="screen">
    <param name="data_path" type="string"  value="$(arg data_path)"/>
    <param name="pcd_file" type="string" value="$(arg pcd_file)"/>
    <param name="odom_file" type="string" value="$(arg odom_file)"/>
    <param name="downsample_size" type="double" value="0.2"/>
    <param name="pcd_name_fill_num" type="int" value="6"/> <!--set 5 for kitti07 and 0 for park-->
    <param name="marker_size" type="double" value="1"/>
    <param name="gap" type="int" value="5"/>
    <param name="start" type="int" value="0"/>

    

  </node>

  <arg name="rviz" default="1"/>
  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz"
    args="-d $(find hba)/rviz_cfg/rosbag.rviz" />
  </group>
</launch>